react/features/base/responsive-ui/actions.js

```diff
@@ -2,6 +2,8 @@
 
 import type { Dispatch } from 'redux';
 
+import { CHAT_SIZE } from '../../chat';
```

Comment:
Please import this from the constants file directly, that will avoid any potential breakage due to circular imports for example.